### ISSUE TITLE
Feature Request: Show current memory address of a modules section

### ISSUE THREAD

**Issue by OCT-Burnz** [processhacker/processhacker#509]
Hello,
i would like to kindly ask for the following feature:

When you open the properties of a module in the modules tab in the general tab a list of all sections is shown. However, only the offset (VA) is shown in the tabs, it would be great to have an extra field that shows the relative addres - or even better, since each entry in the list is already clickable - be able to double click the entry and a memory-view would open at the memory address of the process.

Why this feature ? When i use Scylla to dump a process i usually have to fix imports. Normally i end-up at the .rdata offset of the executable but have to manually calculate the final address.

Thank you for the consideration.

**Comment by dmex** [comment]
> shows the relative addres

You can see the relative address on the Inspect menu

**Comment by OCT-Burnz** [comment]
Inspect General Tab only shows the offset of the section, not the address itself...

**Comment by dmex** [comment]
The memory address for the section doesn't always match actual address because on Windows there are different methods for laying a region of image memory with SEC_COMMIT and SEC_IMAGE.

Tried pasting the addresses into calc and using the +?

![image](https://user-images.githubusercontent.com/1306177/70048394-5048ce00-161e-11ea-9cd2-aa2a03ba7511.png)
![image](https://user-images.githubusercontent.com/1306177/70048409-58087280-161e-11ea-95c7-209efb4e9775.png)

**Comment by OCT-Burnz** [comment]
Yes, currently i'm doing that with Calc. My feature request actually would be showing that address that we calculated in Calc in the table (and be able to copy+paste it). I'm doing those operations multiple times a day and i noticed when you do it 10 times a day it's quite a timesink...

**Comment by OCT-Burnz** [comment]
I changed Scylla to take the address as relative offset instead of a virtual address.

**Comment by dmex** [comment]
> when you do it 10 times a day

This would be an issue in that case. I'll look into passing the address to peviewer.
